.devcontainer/Dockerfile

```
@@ -0,0 +1,22 @@
+from flink:1.17.0
```

Got a linter warning here, complaining about the lowercase keyword. 
```suggestion
FROM flink:1.17.0
```

---

See the comment above and also the teams messages I sent you.